### PR TITLE
Unskip 9 tests on df.getitem

### PR DESCRIPTION
**Pull request by densmirn**

### CODE REVIEW

File: sdc/tests/test_dataframe.py [hunk 1]
```diff
@@ -332,7 +325,7 @@ def test_impl(n):
         self.assertEqual(count_array_REPs(), 0)
         self.assertEqual(count_parfor_REPs(), 0)
 
-    @skip_numba_jit
+    @skip_numba_jit('iadd unsupported')
```
Comment (AlexanderKalistratov) [inline sdc/tests/test_dataframe.py]: what is `iadd`?

Comment (densmirn) [reply sdc/tests/test_dataframe.py]: Actually `np.sum` of `Series` unsupported as I see. `iadd` was mentioned in internal error.

Comment (AlexanderKalistratov) [reply sdc/tests/test_dataframe.py]: do we support `loc` and `iloc` for dataframes?

Comment (densmirn) [reply sdc/tests/test_dataframe.py]: Also no, but the first error of the test run is about `np.sum`. If to exclude call of `np.sum` we get error on `loc`.

**Pull request merged**